policy/modules/kernel/files.fc

```diff
@@ -299,8 +299,8 @@ ifndef(`distro_redhat',`
 /var/run/lock/.*		<<none>>
 
 /run/cockpit/motd	--	gen_context(system_u:object_r:etc_t,s0)
-/run/motd		--	gen_context(system_u:object_r:etc_t,s0)
-/run/motd.d(/.*)?		gen_context(system_u:object_r:etc_t,s0)
+/var/run/motd		--	gen_context(system_u:object_r:etc_t,s0)
+/var/run/motd.d(/.*)?		gen_context(system_u:object_r:etc_t,s0)
```

Comment:
you should escape the `.` as `.` means `anything` with regex: `/var/run/motd\.d(/.*)?`

Reply:
Thanks for catching that, fixing!